src/Shared/PackageManagers/NuGetProjectManager.cs

```diff
@@ -308,26 +226,6 @@ public override async Task<IEnumerable<string>> EnumerateVersions(PackageURL pur
                 return null;
```

Comment:
Should this method be updated to use API from the Nuget library as well?

Reply:
Updating it to use the NuGet.Protocol library.